src/en/about/guide.md

```diff
@@ -1,57 +1,95 @@
+
+Admins are super friendly people who are usually running the server out of their own pocket, so it’s good to get to know them. Many accept donations to cover the running costs and if you’re able to contribute then it’s always appreciated.
+
+## I think I found my new home!
+
+Head to the Login page `(https://your.server/login)` of your server and choose your username and password in the sign up form. You don't need an email to register, but this is the only way to reset your password if you lose it.
+
+Next thing to do is upload your profile picture, give the settings page `(https://your.server/settings)` a good once-over (and do come back to it when you’ve been on Lemmy a week or so just to make any tweaks that might help your experience) and get ready to introduce yourself.
+
+*Some interesting settings worth checking are: your default feed sorting filter between subscribed, local or all and your default sorting type.*
+
+While you can simply enjoy reading Lemmy, the real fun begins as you dive in and start participating yourself by posting, voting and commenting.
+
+To share a post, click the “Create Post” button on the top side of the screen. You’ll be taken to a form where you can enter a URL of the link or upload your image, and input a title and body for the post. You will also need to enter the community where you’d like your post to appear. Before doing so, it’s a good idea to head to that communitie’s page and read the posting guidelines that appear on the right side of the screen. This will ensure you are abiding by the particular ethos of the community. For example, on the [`!privacy@lemmy.ml`](https://lemmy.ml/c/privacy) community, you are not allowed to promote proprietary software.
+
+When all the fields are filled out, click “Create” and your post will go live.
+
+By writing a new post or comment, start typing...
 
 - `@a_user_name` to get a list of usernames.
 - `!a_community` to get a list of communities.
 - `:emoji` to get a list of emojis.
 
-## Sorting
+## A quick tour of the web interface
 
-*Applies to both posts and comments*
+![interface.png](interface.png)
 
-Type | Description
---- | ---
-Active | Trending sort based on the score, and the most recent comment time.
-Hot | Trending sort based on the score, and the post creation time.
-New | Newest items.
-Most Comments | The posts with the most comments.
-New Comments | The posts with most recent comments, IE a forum-style sort.
-Top | The highest scoring items in the given time frame.
+The standard Lemmy interface has a single feed. You can change the content of the feed by using the filters:
 
-For more detail, check the [Post and Comment Ranking details](ranking.md).
+**Subsribed** includes the posts of people on your server and on other servers, all that matters is that it’s people you follow.
```

Comment:
typo on subscribed. Split into two sentences.

> all that matters is that it’s people you follow.

Lemmy actually doesn't have user following, its communities you follow.